lib/matplotlib/tests/test_backend_svg.py

```diff
@@ -208,3 +211,67 @@ def psfont(*args, **kwargs):
     ax.text(0.5, 0.5, 'hello')
     with tempfile.TemporaryFile() as tmpfile, pytest.raises(ValueError):
         fig.savefig(tmpfile, format='svg')
+
+
+def test_gid_data():
+    def draw_graphics(ax):
+        rect = mpatches.Rectangle((0.5, 0.5), 1.5, 1.5, fc='red', alpha=0.5)
+        rect.set_gid("rectangle-of-paint")
+        ax.add_patch(rect)
+        tx = ax.text(0.15, 0.0, "Wet Paint")
+        tx.set_gid("warning-message")
+        ax.set_xlim(0, 5)
+        ax.set_ylim(0, 5)
+        ax.xaxis.set_visible(False)
+        ax.yaxis.set_visible(False)
+        for spine in ax.spines.values():
+            spine.set_visible(False)
+
+    fig = figure.Figure((2, 2))
+    canvas = FigureCanvasSVG(fig)
+    ax = fig.add_subplot(1, 1, 1)
+    draw_graphics(ax)
+    figure_buffer = BytesIO()
+    svg_attribs = {
+        'extra_content': b"""
```

Comment:
It was `bytes` to exercise the `force_textual` function, but if we're instead enforcing pre-encoding everything as text, this isn't necessary.